<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>413</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>电机功率计算</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>所需功率：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>kw</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;由拉深工艺核算变频电机功率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>267</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>拉深变形力：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>0.55</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.60</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.65</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.70</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.75</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.80</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="text">
       <string>0.8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>拉深系数：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>25</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>拉深深度：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>t</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>实际滑块行程次数：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>滑块行程次数：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="text">
       <string>25</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>系数c</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>电机运行系数</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="text">
       <string>1.3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>r/min</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>r/min</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>318</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>压力机效率：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="text">
       <string>0.2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>压边力：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>N</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>拉深垫工作行程</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>压力机结构形式：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>送料模式：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>单级传动快速压力机</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>多级传动慢速压力机</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>带拉深垫压力机</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_3">
      <item>
       <property name="text">
        <string>手工送料</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>自动送料</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>jisuan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>shzhixiaolv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_3</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>shzhixiaolv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>bianhuaxisu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_9</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>dianjiyunxing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>jisuan()</slot>
  <slot>shzhixiaolv()</slot>
  <slot>slot1()</slot>
  <slot>bianhuaxisu()</slot>
  <slot>dianjiyunxing()</slot>
 </slots>
</ui>
